CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # node-google-dfp-wrapper
 
+### 1.3.0
+- Adds `getCriteriaValues` for querying criteria values without extracting id.
+- Adds `getCriteriaValueId` for querying criteria values and extracting id.
```

Comment:
I guess this might not be clear, the id will always be included, what's different is whether it's the only field or if there will be other fields.